include/benchmark/benchmark.h

```diff
@@ -667,6 +667,12 @@ class State {
   BENCHMARK_DEPRECATED_MSG("use 'range(1)' instead")
   int64_t range_y() const { return range(1); }
 
+  BENCHMARK_ALWAYS_INLINE
+  int num_threads() { return threads; }
```

Comment:
hm. i'd really like to name this `threads()` right away to follow https://google.github.io/styleguide/cppguide.html#Function_Names 

given i just released v1.5.6 i think we can make this API change and the next release will be v1.6.0.

anyone building against HEAD should be flexible about API changes i think.

Reply:
My concern is it'd break tensorflow_serving  if we delete `int threads` and replace it with `int threads()` It'd be better (less headache for me, mostly) if there were intermediate states where I could migrate people gradually without forcing the updates on them ("take this or you'd break" kind of change is really stressful).

Reply:
does tensorflow_serving link to HEAD? i would hope most clients are pinned to a version or aware that they need to fix stuff against HEAD...

Reply:
tensorflow_serving depends on tensorflow (against a specific version/hash commit), which then links to a specific benchmark version (`64cb55e91067860548cb95e012a38f2e5b71e026` right now).

But actually, I've done a quick search, only two tests are using this - I think it's small enough that it's not a problem.

Renamed the function to `threads()` and make the field private.